recipes/default.rb

```diff
@@ -17,3 +17,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+case node['platform_family']
+when "windows"
+	include_recipe 'vcruntime::vc6'
```

Comment:
I am relatively new to chef, i will research cookstyle and see to replace tabs with spaces. ive always coded with tabs, never really understood the difference. its just easier to indent sections in bulk using tabs for me anyway.